docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -96,3 +99,84 @@ encode_tx       [type=ethabiencode
 
 submit_tx  [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 ```
+
+**Multi word example**
+
+```sol
+contract MyClient is ChainlinkClient {
+    function doRequest(uint256 _payment) public {
+        Chainlink.Request memory req = buildChainlinkRequest(specId, address(this), this.fulfill.selector);
+        req.add("urlUSD", "https://datafeed.xyz/ethusd");
+        req.add("pathUSD", "data");
+        req.add("urlEUR", "https://datafeed.xyz/etheur"));
+        req.add("pathEUR", "data");
+        req.add("urlJPY", "https://datafeed.xyz/ethjpy");
+        req.add("pathJPY", "data");
+        requestOracleData(req, _payment); // MWR API.
+    }
+
+    function fulfill(bytes32 requestID, uint256 usd, uint256 eur, uint256 jpy) public {
+        // ...
+    }
+}
+```
+
```

Comment:
Can we get a short description sentence to explain the difference in the job between single and multi?